sycl/test/regression/constexpr-fp16-numeric-limits.cpp

```diff
@@ -0,0 +1,29 @@
+// RUN: %clangxx -fsycl-device-only -fsyntax-only -Xclang -verify %s
+// expected-no-diagnostics
+#include <CL/sycl.hpp>
+
+int main() {
+  cl::sycl::queue Q;
+  Q.submit([&](cl::sycl::handler &CGH) {
+    CGH.single_task<class Test>([=]() {
+#ifdef __SYCL_DEVICE_ONLY__
```

Comment:
`-fsycl-device-only` enables this macro.

Reply:
Thanks, I've simplified the test